cosmos/lib/cosmos/script/storage.rb

```diff
@@ -0,0 +1,160 @@
+# encoding: ascii-8bit
+
+# Copyright 2022 Ball Aerospace & Technologies Corp.
+# All Rights Reserved.
+#
+# This program is free software; you can modify and/or redistribute it
+# under the terms of the GNU Affero General Public License
+# as published by the Free Software Foundation; version 3 with
+# attribution addendums as found in the LICENSE.txt
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# This program may also be used under the terms of a commercial or
+# enterprise edition license of COSMOS if purchased from the
+# copyright holder
+
+require 'net/http'
+
+module Cosmos
+  module Script
+    private
+
+    # Get a handle to access a target file
+    #
+    # @param path [String] Path to a file in a target directory
+    # @param binary [Boolean] Whether the file is binary or not
+    # @return [Hash|nil]
+    def get_target_file(path, original: false, scope: $cosmos_scope)
+      # Create Tempfile to store data
+      file = Tempfile.new('target', binmode: true)
```

Comment:
binmode: true basically works for everything?

Reply:
Yes, we don't ever want to convert line endings